lightly_studio_view/src/routes/datasets/[dataset_id]/videos/+page.svelte

```diff
@@ -75,7 +100,30 @@
                 {#snippet item({ index, style })}
                     {#if items[index]}
                         {#key items[index].sample_id}
-                            <div {style}>
+                            <div
```

Comment:
We’re currently building similar components for the samples, annotations, and videos grid views. To reduce code duplication and improve maintainability, let’s consider creating a reusable component that can serve all three views.